unittests/test_cli.py

```diff
@@ -473,6 +473,15 @@ def test_verbosity(self):
         assert 'Traceback' not in stderr
         assert 0 == returncode
 
+    def test_verbosity_with_check(self):
+        self.more_options = ['-vvvvv']
+        self.checkpath = ['unittests/resources/checks/hellocheck.py']
+        returncode, stdout, stderr = self._run_reframe()
+        assert '' != stdout
+        assert 'Traceback' not in stdout
+        assert 'Traceback' not in stderr
+        assert 0 == returncode
```

Comment:
You should add an assert here that no `--- Logging error ---` appears in the output. We have this in another unit test as well.